runtime/debugger.c

```diff
@@ -134,11 +126,7 @@ static void open_connection(void)
       strerror (errno)
```

Comment:
Under Windows, if `WSASocket` or `connect` fail, `errno` is not set to a meaningful error code, so the error message will be strange.  But this was already the case with the old implementation.

Reply:
Would you prefer dedicated error handling on the Windows side, or just setting `errno = 0` and voilà?